stresso/src/test/java/stresso/TrieMapRedIT.java

```diff
@@ -35,102 +28,107 @@
 import stresso.trie.Print;
 import stresso.trie.Unique;
 
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 /**
  * Tests Trie Stress Test using MapReduce Ingest
  */
+@SuppressWarnings("ALL")
```

Comment:
Why add this?

Reply:
my formatter different yours. this line code false. i’ll delete. sorry.

Reply:
Intellij add getting at inspected. Only classes, methods, variable declarations, parameters and packages may be annotated. Therefore, you cannot use SuppressWarnings("all") on an if statement.